docs/deployment/google_cloud_run.mdx

```diff
@@ -0,0 +1,50 @@
+---
+id: gcp-cloud-run
+title: Cloud Run
+description: Prepare your crawler to run in Cloud Run on Google Cloud Platform.
+---
+
+import ApiLink from '@site/src/components/ApiLink';
+
+import CodeBlock from '@theme/CodeBlock';
+
+import GoogleCloudRun from '!!raw-loader!./code_examples/google/cloud_run_example.py';
+
+For deploying crawlers based on headless browsers or applications requiring more granular control than [GCP Cloud Functions](./gcp-cloud-run-functions) can provide, [Cloud Run](https://cloud.google.com/run) is the recommended solution for Python-based Crawlee applications.
+
+GCP Cloud Run allows you to deploy using Docker containers, giving you full control over your environment and the flexibility to use any web server framework of your choice, unlike Cloud Functions which are limited to [Flask](https://flask.palletsprojects.com/en/stable/). Cloud Run containers are spun up on-demand, so you're only billed for the time it takes to process each request, while also providing a better developer experience by allowing you to test your containers locally before deploying.
+
+## Preparing the project
+
+We'll prepare our project using [Litestar](https://litestar.dev/) and the [Uvicorn](https://www.uvicorn.org/) web server. The HTTP server handler will wrap the crawler to communicate with clients. Because the Cloud Run platform sees only an opaque Docker container, we have to take care of this bit ourselves.
+
+:::info
+
+GCP passes you an environment variable called `PORT` - your HTTP server is expected to be listening on this port (GCP exposes this one to the outer world).
+
+:::
+
+<CodeBlock className="language-python">
+    {GoogleCloudRun.replace(/^.*?\n/, '')}
+</CodeBlock>
+
+
+:::tip
+
+Always make sure to keep all the logic in the request handler - as with other FaaS services, your request handlers have to be **stateless.**
+
+:::
+
+## Deploying to GCP
```

Comment:
In Cloud Run we have "Deploying to Google Cloud Platform" title, could we unify them please?